aws-auth-cognito/src/main/java/com/amplifyframework/auth/cognito/exceptions/invalidstate/SignedInException.kt

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+package com.amplifyframework.auth.cognito.exceptions.invalidstate
+
+import com.amplifyframework.auth.exceptions.InvalidStateException
+
+/**
+ * Auth exception caused attempting an operation that is not allowed while signed in
+ * @param message Explains the reason for the exception
+ * @param recoverySuggestion recoverySuggestion Text suggesting a way to recover from the error being described
+ */
+open class SignedInException(
+    message: String = "There is already a user in signedIn state.",
```

Comment:
Made this change to messaging https://github.com/aws-amplify/amplify-android/pull/2000/files#diff-97fa2fb9d98422fe1811926c96a20193d5525f50086bff8e8aac5991dae7207aR96, since "signedIn state" is internal and sign in can also happen by calling `signInWithWebUI`.

Reply:
Fixing